json-schema/json-schema/src/test/scala/endpoints/algebra/JsonSchemasTest.scala

```diff
@@ -16,6 +16,14 @@ trait JsonSchemasTest extends JsonSchemas {
       field[String]("name", Some("Name of the user")) zip
       field[Int]("age")
     ).invmap((User.apply _).tupled)(Function.unlift(User.unapply))
+
+    val schema2: JsonSchema[User] = (
+      emptyRecord zip
+      field[String]("name", Some("Name of the user")) zip
+      field[Int]("age")
+    )
+      .invmap[(String, Int)](p => (p._1._2, p._2))(p => (((), p._1), p._2))
```

Comment:
ideally would like to use `.tupled[User]`, but `RecordGenericOps` are only available in `json-schema-generic`